style.css

```diff
@@ -453,32 +484,31 @@ textarea {
   left: 0;
   right: 0;
   background-color: #c1c7d0d7;
+  width: 109vw;
+  height: 109vh;
+}
+
+.display-centered {
+  display: none;
 }
 
 .card {
   position: fixed;
-  top: 5%;
-  bottom: 5%;
-  left: 15%;
-  right: 15%;
   background-color: white;
   border-radius: 5px;
   overflow: hidden;
+  width: 100vw;
 }
 
-.card-header {
-  display: flex;
-  flex-direction: row-reverse;
-  padding: 5px;
-}
-
-.card-header .close-button {
-  margin-right: 5px;
-  margin-top: 5px;
+.card-image {
+  position: absolute;
+  height: 616px;
+  left: 24px;
+  top: 78px;
 }
 
 .card-body {
-  height: 90%;
+  height: 80vh;
   display: flex;
   flex-direction: column;
   padding-left: 10px;
```

Comment:
- [ ] In the process of implementing the previous suggestions, the popup cards are no longer aligned correctly to the center. Kindly see the image below and implement the changes 
## Figma Design
![image](https://user-images.githubusercontent.com/46839548/218278784-65935ba8-467d-42ce-b5f3-329afe9b5bd5.png)
## Your Design
![image](https://user-images.githubusercontent.com/46839548/218278664-b5460abf-0d09-4f72-800a-9c24fa622a9e.png)